components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksInterceptor.kt

```diff
@@ -44,7 +44,8 @@ class AppLinksInterceptor(
     private val alwaysAllowedSchemes: Set<String> = setOf("mailto", "market", "sms", "tel"),
     private val alwaysDeniedSchemes: Set<String> = setOf("javascript", "about"),
     private val launchInApp: () -> Boolean = { false },
-    private val useCases: AppLinksUseCases = AppLinksUseCases(context, launchInApp)
+    private val useCases: AppLinksUseCases = AppLinksUseCases(context, launchInApp),
+    private val launchFromInterceptor: Boolean = false
```

Comment:
nit: Let's add some docs for this new parameter :)

Reply:
oops, thanks for catching this.